### DISCUSSION TITLE
Set heatmap color based on Y-axis

### DISCUSSION THREAD

**Discussion by djahandarie** [grafana/grafana#45634] [Dashboards, Panels & Visualizations]
I currently have a heatmap which has latency on the Y-axis, and time on the X-axis.

I would like the value to control the opacity (already possible by toggling the display mode to "opacity"), and **then for the color to be dynamic based on the *Y-axis*** (ideally some thresholds I could set, like <100ms green, <1s yellow, >1s red... but one of the spectrums would also be fine TBH).

This would allow for a graph that would "have a little red" when a little of something really bad happens, and "a lot of yellow" when a lot of something moderately bad happens, etc. etc., which is very intuitive for most people. Right now heatmaps are not very understandable for laypeople (because they will barely even notice some faint colors at the top of the graph, but that is very bad depending on the situation).

(cc @ryantxu because I believe you are working on a new heatmap right now)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
